firebase-appdistribution/api.txt

```diff
@@ -1,75 +1 @@
 // Signature format: 2.0
-package com.google.firebase.appdistribution {
```

Comment:
I wonder if moving this file will impact the reference docs. If we move the api.txt file - reference docs might be generated under the package name `firebase-appdistribution-api`